Scripts/Tracking/Velocity/VelocityTrackerProcessor.cs

```diff
@@ -46,7 +55,7 @@ public override Vector3 GetVelocity()
         /// The reported angular velocity on the first active <see cref="VelocityTracker"/>.
         /// </summary>
         /// <returns>The current angular velocity.</returns>

```

Comment:
This XML docs block should just be `<inheritdoc />`.

Reply:
same here, the docs are specific to the method